app/assets/javascripts/give_time.js.coffee

```diff
@@ -0,0 +1,18 @@
+jQuery.validator.addMethod "either-hours-minutes-informed", ((value, element) ->
+  $("#transfer_hours").val() or $("#transfer_minutes").val()
+), "Hours or minutes should be informed"
+
+giveTimeReadyFn = () ->
+  config =
+    submitHandler: (form) ->
+      amount = $("#transfer_hours").val() * 3600 + $("#transfer_minutes").val() * 60
+      $("#transfer_amount").val(amount)
+
+      if amount > 0
+        form.submit()
+      else
+        $(form).find('.form-actions .error').removeClass('invisible').show()
+
+  $( "#new_transfer" ).validate(config)
+
+$( document ).ready giveTimeReadyFn
```

Comment:
This file `app/assets/javascripts/give_time.js.coffee` doesn't exist in develop, I think it was deleted after you start working in your fork

Reply:
Fixed, the file has been deleted